install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+set -e
+
+temp_dir=$(mktemp -d /tmp/xh.XXXXXXXX)
+cd "$temp_dir"
+
+if [ "$(uname -s)" = "Darwin" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="x86_64-apple-darwin"
+elif [ "$(uname -s)" = "Linux" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="x86_64-unknown-linux-musl"
+elif [ "$(uname -s)" = "Linux" ] && [ "$(uname -m)" = "amd64" ]; then
```

Comment:
Can't hurt to leave this in, but is `uname -m` ever `amd64` on Linux? `x86_64` seems standard.

Reply:
It was something I originally copied from zoxide's [install script](https://github.com/ajeetdsouza/zoxide/blob/main/install.sh#L32) but I agree that it is not needed 👍